iree/compiler/InputConversion/MHLO/Passes.h

```diff
@@ -19,8 +20,27 @@ namespace iree_compiler {
 // Performs input legalization for specific combination of input dialects.
 void buildMHLOInputConversionPassPipeline(OpPassManager &passManager);
 
+// Performs some cleanup activities on programs that may have originated from
+// an XLA import (or made to interop with it). This involves:
+//   - Converting XLA control flow to SCF
+//   - Converting SCF control flow to CFG
+//   - Flatten tuples in CFG
+//   - Canonicalize
```

Comment:
Nit: consistent part of speech

```suggestion
//   - Flattening tuples in CFG
//   - Canonicalizing
```